config/crd/bases/pipelines.openshift.io_gitopsservices.yaml

```diff
@@ -1,9 +1,11 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
Looks like your local version of controller-gen is old. Can you check this?

Reply:
updated